src/react-styleguide/src/App.js

```diff
@@ -15,6 +17,14 @@ class App extends Component {
           <CodeBlock code={ButtonExampleCode} />
           <PropTable rows={ButtonPropsTable} />
         </div>
+
+        <div className='mw8 center'>
+          <Header title='Inputs' />
+          <InputExamples />
+          <CodeBlock code={InputExampleCode} />
+          <PropTable rows={InputPropsTable} />
+        </div>
```

Comment:
Could map over each style guide component to output Header, Examples, CodeBlock, and PropTable if you want to reduce the repetition here.